[camera] Allow logical cameras to use all physical cameras via zoom on android 11+ (#6150)
flutter/plugins: packages/camera/camera_android/CHANGELOG.md

> ## 0.10.0+1

Reviewer: Nit: maybe mention deleting `CameraZoom` here, as well?

> * Fixes issues with Virtual cameras hiding physical cameras in android 11+.

Reviewer: Nit, address zoom issue and capitalize Android, something like:

```suggestion
* Fixes zoom computation for virtual cameras hiding physical cameras in Android 11+.
```
